Merging to main without bump fails CI.

For example:
```
Run cz bump --changelog --yes
bump: release 0.10.5 → 0.10.5

Automatically generated by Commitizen.

tag to create: v0.10.5
[NO_COMMITS_TO_BUMP]
The commits found are not eligible to be bumped

Error: Process completed with exit code 21
```
We probably shouldn’t fail CI here. However, looks like `cz` doesn’t support this on [bump](https://commitizen-tools.github.io/commitizen/bump/) 🤔

--- a/.github/workflows/release.yaml
+++ b/.github/workflows/release.yaml
@@ -61,7 +61,7 @@ jobs:
         git config --list --global # For debug purposes.
 
     - name: Create changelog and bump
-      run: cz bump --changelog --yes
+      run: cz --no-raise 21 bump --changelog --yes
 
     - name: Push the release
       run: |

--- a/.github/workflows/release.yaml
+++ b/.github/workflows/release.yaml
@@ -60,6 +60,9 @@ jobs:
         git config --global user.email "$USER_EMAIL"
         git config --list --global # For debug purposes.
 
+    # In some cases a user may merge commits that don't cause a version bump, which causes commitizen
+    # to fail with error code 21 (NoneIncrementExit). Thus we silence that particular error to avoid
+    # failing this job: https://commitizen-tools.github.io/commitizen/bump/#avoid-raising-errors
     - name: Create changelog and bump
       run: cz --no-raise 21 bump --changelog --yes
 

